keps/sig-node/3570-cpumanager/README.md

```diff
@@ -156,11 +228,12 @@ type CPUSet map[int]struct{} // set operations and parsing/formatting helpers
 type CPUTopology // convenient type for querying and filtering CPUs
 ```
 
-#### Configuring the CPU Manager
+### Configuring the CPU Manager
```

Comment:
As `CPUManagerPolicyOptions` allows us to tweak the behavior of static policy, we should probably add some information about it here. Just providing a list of policy options and adding references to the relevant KEPs should be sufficient:

1.  https://github.com/fromanirh/enhancements/tree/master/keps/sig-node/2625-cpumanager-policies-thread-placement
2. https://github.com/fromanirh/enhancements/tree/master/keps/sig-node/2902-cpumanager-distribute-cpus-policy-option
3. https://github.com/fromanirh/enhancements/tree/master/keps/sig-node/3327-align-by-socket

Reply:
good point, done.